langkit/sentiment.py

```diff
@@ -19,16 +33,10 @@ def init(lexicon: Optional[str] = None):
         nltk.download(lexicon)
         _nltk_downloaded = True
     _sentiment_analyzer = SentimentIntensityAnalyzer()
-
-
-@register_metric_udf(col_type=String)
-def sentiment_nltk(text: str) -> float:
-    if _sentiment_analyzer is None:
-        raise ValueError(
-            "sentiment metrics must initialize sentiment analyzer before evaluation!"
+    for column in [lang_config.prompt_column, lang_config.response_column]:
+        register_dataset_udf([column], udf_name=f"{column}.sentiment_nltk")(
+            sentiment_nltk
         )
-    sentiment_score = _sentiment_analyzer.polarity_scores(text)
-    return sentiment_score["compound"]
```

Comment:
Can we preserve the `sentiment_nltk `function, and create a wrapper for the dataset_udf shape, so that we don't break users who called sentiment_nltk